CMakeLists.txt

```diff
@@ -3,7 +3,7 @@
 # Copyright (c) 2018-2020 INRIA
 #
 
-CMAKE_MINIMUM_REQUIRED(VERSION 2.6)
```

Comment:
Why did you choose this version?

Reply:
because of jrl-umi3218/jrl-cmakemodules#338, as documented in the commit description